samples/snippets/src/main/java/com/example/servicedirectory/EndpointsCreate.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.servicedirectory;
+
+// [START servicedirectory_create_endpoint]
+
+import com.google.cloud.servicedirectory.v1beta1.Endpoint;
+import com.google.cloud.servicedirectory.v1beta1.RegistrationServiceClient;
+import com.google.cloud.servicedirectory.v1beta1.ServiceName;
+import java.io.IOException;
+
+public class EndpointsCreate {
+
+  public static void createEndpoint() throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "your-project-id";
+    String locationId = "your-region";
+    String namespaceId = "your-namespace";
+    String serviceId = "your-service";
+    String endpointId = "your-endpoint";
+    createEndpoint(projectId, locationId, namespaceId, serviceId, endpointId);
+  }
+
+  // Create a new endpoint.
+  public static void createEndpoint(
+      String projectId, String locationId, String namespaceId, String serviceId, String endpointId)
+      throws IOException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
+    try (RegistrationServiceClient client = RegistrationServiceClient.create()) {
+
+      // The service to create the endpoint in.
+      ServiceName parent = ServiceName.of(projectId, locationId, namespaceId, serviceId);
+
+      // Optionally set an IP address and port for the endpoint.
+      String address = "10.0.0.1";
+      int port = 443;
+
+      // The endpoint to create, with fields filled in.
+      Endpoint endpoint = Endpoint.newBuilder().setAddress(address).setPort(port).build();
```

Comment:
Can completely inline. 
```suggestion
      // The endpoint to create, with fields filled in.
      // Optionally set an IP address and port for the endpoint.
      Endpoint endpoint = Endpoint.newBuilder().setAddress("10.0.0.1").setPort(443).build();
```

Reply:
Done